### ISSUE TITLE
Search keyword triggers on Firefox even when option is off

### ISSUE THREAD

**Issue by mlissner** [rugk/awesome-emoji-picker#80]
## Bug description
STR:

0. Disable search bar interaction
1. Press CTRL+L to put your cursor in the URL bar (clicking would work too)
2. Search for "emoji slapping forehead" (without quotes)
3. Press enter
4. You're at the settings for the emoji picker! 

Weird! 

### Expected behavior
This should just do the query, since before now I knew nothing about the search bar feature (and it's off)

## System
Browser and version: Firefox 
Add-on version: 1.2

**Comment by rugk** [comment]
Yes, this is expected, because I can only statically define [the keyword search](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/user_interface/Omnibox). Thus, I either can ignore that and no search will trigger.
So to not do nothing, I at least do show the error page.

Thus I cannot really solve that.

**Comment by mlissner** [comment]
Makes sense. It doesn't really go to an error page though so much as a settings page. Maybe just tweak that? 

(I'll say from a privacy perspective I was pretty surprised when my search got sent to you. I have an extension I manage with a search bar feature too though, so perhaps that's just par for the course. The difference though is that my search keyword is more specific — it's a more uncommon word — so it's pretty unlikely people would stumble upon the error like I did here. OTOH, sending you this query about emojis doesn't tell you much.)

Anyway, a simple fix might be a better error page?

**Comment by rugk** [comment]
Okay, I can see the issue with privacy. I deliberately choose a common (and short) word, so it is easy to search for.

The idea behind opening the options page was that you see the feature at the bottom there and can enable it. And do know what triggered that mhh...

But showing an additional error may not be a bad idea. (as it is not _that_ obvious) Anyway on the options page, IMHO, then.

**Comment by rugk** [comment]
Another workaround would be to trigger the default search. That, [is actually already implemented](https://github.com/rugk/awesome-emoji-picker/blob/1dcd58dd5fb7689a552a3511640eb03982d223e0/src/background/modules/OmniboxSearch.js#L106-L129), and you only need to grant the add-on the "search" permission for that.
However, I did not want to request that permission, because I don't need it for anything else. Also, [given it cannot be requested at runtime, because it is not an optional permission](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/optional_permissions#API_permissions), that would even trigger at the next add-on update and make it the only permission to be explicitly shown at runtime.

**Comment by mlissner** [comment]
What's worse is that when the upgrade rolled out, all your current users would have to agree to it. When we've done that in the past we load hoards of users overnight. Maybe they weren't our top users, but still, it hurt.

> The idea behind opening the options page was that you see the feature at the bottom there and can enable it. And do know what triggered that mhh...

Yeah, I figured it out, but I'm a developer, FF fan, and am even familiar with extensions. Anyway, thanks for thinking about this. And thanks for the extension. I use it a lot.

**Comment by rugk** [comment]
> What's worse is that when the upgrade rolled out, all your current users would have to agree to it. 

What do you mean. The "search" permission? Yes, that's what I've said and did not want to do.
One may always do so in a later update though.
